rspec-core.gemspec

```diff
@@ -47,7 +47,8 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency "cucumber", "~> 1.3"
   s.add_development_dependency "minitest", "~> 5.3"
-  s.add_development_dependency "aruba",    "~> 0.6.2" # 0.7 is broken on ruby 1.8.7
+  s.add_development_dependency "aruba",    "~> 0.14.8"
+  s.add_development_dependency "contracts", "< 0.16" # 0.16 does not work with Ruby 1.8.7
```

Comment:
What does the extra dependency do?

Reply:
`aruba` depends on `contracts '~> 0.9'`, and apparently `0.16` is no good, but `0.15` is ok.

Reply:
I'm going to move this to the Gemfile so the current ruby version can be checked.